specs/mediawiki/v1/content.yaml

```diff
@@ -445,6 +445,69 @@ paths:
       x-backend-request:
         uri: /{domain}/sys/page_revisions/rev/{revision}
 
+  /{module:page}/wikitext/{title}{/revision}:
+    post:
+      tags:
+        - Page content
+      description: >
+        Save a new revision of a page
+
+        For new pages, the `revision` parameter should be left empty.
+        For editing existing pages, it should contain the revision ID
+        the edit is derived from.
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Unstable)
+      parameters:
+        - name: title
+          in: path
+          description: The page title.
+          type: string
+          required: true
+        - name: revision
+          in: path
+          description: The revision of the page the edit is based on.
+          type: string
+        - name: text
```

Comment:
Maybe call this 'wikitext', so that we could add additional parts later? An example would be a save of wikitext + page metadata.